test-suite/djinni/test.djinni

```diff
@@ -49,3 +49,8 @@ test_helpers = interface +c {
 empty_record = record {
     
 }
+
+# Extended record
```

Comment:
I'd suggest breaking this out into its own file like extended_record.djinni
Also you should add an interface which uses this record, and another record which contains this record, to make sure the include is correct in all possible contexts.